sos/report/plugins/__init__.py

```diff
@@ -1368,10 +1409,22 @@ def _collect_cmd_output(self, cmd, suggest_filename=None,
             # Exit here if the command was not found in the chroot check above
             # as otherwise we will create a blank file in the archive
             if result['status'] in [126, 127]:
-                return result
+                if self.manifest:
+                    self.manifest.commands.append({
+                        'command': cmd.split(' ')[0],
+                        'parameters': cmd.split(' ')[1:],
+                        'exec': cmd,
+                        'filepath': None,
+                        'return_code': result['status'],
+                        'run_time': time() - start,
+                        'tags': tags
+                    })
+                    return result
+
+        end = time() - start
```

Comment:
code de-duplication: isnt it worth having:

    end = time() - start
    manifest_cmd = { 'command': .. .. .. 'tags': tags }
    if <command_not_found>:
        if self.manifest:
            self.manifest.commands.append(manifest_cmd)
        return result
    ..
    if manifest:
        manifest_cmd['filepath'] = outfn
        self.manifest.commands.append(manifest_cmd)

(maybe with better name of `manifest_cmd` variable)

Reply:
Also, `end` variable has confusing name (cf. `'run_time': end,`) - could it be renamed to e.g. `runtime`?

Reply:
Ack to both.